core/testing/go.mod

```diff
@@ -0,0 +1,18 @@
+module cosmossdk.io/core/testing
```

Comment:
I see a new go.mod, so we'll need to follow those tasks here too: https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#gomod